py/desiutil/test/test_bitmask.py

```diff
@@ -105,6 +106,29 @@ def test_str(self):
             self.assertEqual(bitmask[name].bitnum, self.ccdmask[name].bitnum)
             self.assertEqual(bitmask[name]._extra, self.ccdmask[name]._extra)
 
+    def test_uint64(self):
+        _bitdefs = dict(ccdmask=list())
+        _bitdefs['ccdmask'].append( ['BAD',   0, "badness"] )
+        _bitdefs['ccdmask'].append( ['HOT',   1, "hothot"] )
+        _bitdefs['ccdmask'].append( ['TEST', 16, "testing"] )
+
```

Comment:
I changed the decorator to skip the test under python 2.x.  I had originally flagged it as expectedFailure, but then it unexpectedly succeeded under python 3.5.  I don't see an obvious way to conditionally flag something as expectedFailure.